param/parameterized.py

```diff
@@ -211,14 +211,21 @@ def _f(*args, **kwargs):
 
 @accept_arguments
 def depends(func, *dependencies, **kw):
+    """
+    depends allows annotating a method on a Parameterized class to
+    express dependencies on Parameter or Parameter metadata on the
+    same class or one of its Parameters if it is also Parameterized.
```

Comment:
Hmm; confusing.  Maybe:

```
    Annotates a Parameterized method to express its dependencies.
    The specified dependencies can be either be Parameters of this
    class, or Parameters of subobjects (Parameterized objects that
    are values of this object's parameters).  Dependencies can either
    be on Parameter values, or on other metadata about the Parameter.
```

Reply:
BTW, is there a way to do multi-line replacements in GitHub's new Suggestions feature?  So far I can only see how to edit a single line.

Reply:
Guess not.